webapp/src/main/java/com/box/l10n/mojito/entity/Repository.java

```diff
@@ -91,7 +91,7 @@ public class Repository extends AuditableEntity {
 
   @OneToMany(mappedBy = "repository", fetch = FetchType.LAZY)
   @NotAudited
-  //  @JsonManagedReference
+  @JsonManagedReference
```

Comment:
@JsonBackReference added to Branch in the spring migration broke the screen shot upload